components/brave_wallet_ui/panel/wallet_panel_api_proxy.ts

```diff
@@ -41,6 +41,10 @@ class WalletPanelApiProxy extends WalletApiProxy {
   cancelConnectToSite (origin: string, tabId: number) {
     this.panelHandler.cancelConnectToSite(origin, tabId)
   }
+
+  closePanelOnDeactivate (close: boolean) {
+    this.panelHandler.closePanelOnDeactivate(close)
+  }
```

Comment:
This style is deprecated, just call `apiProxy.panelHandler.closePanelOnDeactivate(close)` directly from where you need to call it.

Reply:
done